src/backend/tools/files.py

```diff
@@ -0,0 +1,95 @@
+from typing import Any, Dict, List
+
+from pypdf import PdfReader
```

Comment:
Should we use the langchain tools?

Reply:
Yeah, we were using the langchain version of the pdf reader before, but I was not sure if we should keep using their packages or the actual module. Wdyt?